ParseTree/ParseTree/InputHandler.cs

```diff
@@ -0,0 +1,240 @@
+// <copyright file="InputHandler.cs" company="Chernoshchokaya Evgenia">
+// Copyright (c) Chernoshchokaya Evgenia. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// </copyright>
+
+namespace ParseTree;
+
+/// <summary>
+/// Works with strings and brings them to the desired form.
+/// </summary>
+public class InputHandler
+{
+    /// <summary>
+    /// Checks input for validity.
+    /// </summary>
+    /// <param name="input">String to check.</param>
+    /// <returns>A value indicating whether the string is valid.</returns>
+    public static bool CheckInput(string input)
+    {
+        if (input.Length < 3 || input[0] != '(' || input[^1] != ')')
+        {
+            return false;
+        }
+
+        int charPosition = 1;
+        if (!IsOperator(input[charPosition]))
+        {
+            var number = string.Empty;
+            for (; input[charPosition] != ')'; charPosition++)
+            {
+                number += input[charPosition];
+            }
+
+            int intNumber;
+            return int.TryParse(number, out intNumber) && charPosition == input.Length - 1;
```

Comment:
```suggestion
            return int.TryParse(number, out int _) && charPosition == input.Length - 1;
```
Нам же само значение не нужно

Reply:
Не поправили